### DISCUSSION TITLE
[Feature] Photos and videos from shared albums to appear on timeline

### DISCUSSION THREAD

**Discussion by lmgithu** [immich-app/immich#10425] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi,

I would like to suggest a feature that could be very handy.

Problem statement: Many people share photos because eg. family members use different accounts where they upload their photos. After uploading the photos themselves they can create a shared album. The problem is, that these photos will never make ones timeline. Eg. if i want to view my family members’ photos i always need to go to the shared albums one by one.

Solution: Add an option to show shared photos on the timeline too chronologically. I think it would be a better experience.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7047

**Comment by Antiheld86** [comment]
https://github.com/immich-app/immich/discussions/7047#discussioncomment-14902559

**Comment by TotalKrill** [comment]
Yeah, I also wish I could get the albums on my timeline, I usually create an album and let people upload their photos from an event. Sometimes I really wish I could add individual photos from the album to my timeline, or sometimes ( seldom ) the entire album. 

Also sometimes I wish I could remove photos from my timeline, or all photos that are in an entire album
